CMakeLists.txt

```diff
@@ -95,6 +106,32 @@ qsk_setup_install()
 add_subdirectory(src)
 add_subdirectory(skins)
 
+include(CMakePackageConfigHelpers)
+
+# write QSkinnyConfigVersion.cmake file for cmake import
+write_basic_package_version_file(
+    ${CMAKE_BINARY_DIR}/_QSkinny/${PACKAGE_NAME}ConfigVersion.cmake
+    VERSION ${PACKAGE_VERSION}
+    COMPATIBILITY AnyNewerVersion)
+
+# Copy QSkinnyConfig.cmake to build dir
+configure_file(${QSK_CMAKE_DIR}/${PACKAGE_NAME}Config.cmake
+    ${CMAKE_BINARY_DIR}/_QSkinny/${PACKAGE_NAME}Config.cmake
+    COPYONLY)
+
```

Comment:
@lgtmak  Why do we need to copy it into the build directory? Is it only for the `qsk_update_package_config_file` stuff?